### ISSUE TITLE
Dashboard's widgets are not accessible with keyboard. Missing success criterion WCAG 2.1 - 2.1.1 + 2.4.7

### ISSUE THREAD

**Issue by tassoman** [piwik/piwik#20997]
## Context

This issue maybe related to #20996. When you land into the Dashboard, you have a "skip" menu which brings you to the widgets. Keyboard navigation through the dashboard's widgets is inconsistent and you get lost. 

Actually, dashboard widgets are only browsable with a mouse. [Understanding of success criterion 2.1 keyboard accessible](https://www.w3.org/Translations/WCAG21-it/#keyboard-accessible)

## Expected Behavior

While tabbing between widgets, there [must be a visual feedback of focus location](https://www.w3.org/Translations/WCAG21-it/#focus-visible). (Success criterion 2.4.7).  
While focusing the widget, you should be able to reach its controls (Success Criterion 2.1.1).  
To assert minimum level of accessibility, at least export button should be accessible, so that visually impaired people can download a table representation of the data and use in their spreadsheet software.

## Current Behavior

When you land into the Dashboard, you have a "skip" menu who brings you to the widgets. Keyboard navigation thought the dashboard's widget is inconsistent and you get lost.

When you finally can reach a widget, you can't handle its control menus: settings (cog icon), visualisation (table icon), export (arrow icon), and you can't select number of results.

There's only a small evidence when you tab into a data-table widget, who gets little outlined. if you try to select the widget. Nothing happens, either.

## Steps to Reproduce (for Bugs)

1. Open [Demo Dashboard](https://demo.matomo.cloud)
2. Tab to Skip to content
3. Select by _using enter_ key
4. Tab around trying to reach "channels datatable widget"

If you manage to reach that widget, you can see a thin outline. Press enter to select the widget but nothing happens. You can't reach controls buttons either.

## Your Environment

* Matomo Version: 4.x
* PHP Version: 8.1
* Browser: any
* Operating System: any
